cylc/flow/xtriggers/wall_clock.py

```diff
@@ -19,11 +19,13 @@
 from time import time
 
 
-def wall_clock(trigger_time=None):
+def wall_clock(trigger_time=None, sequential=True):
```

Comment:
This means `wall_clock` xtrigger does not respect the `[scheduling]sequential xtriggers default` setting. Was that intentional?

Reply:
Correct, and intentional (`[scheduling]sequential xtriggers default = False` is the default`).  
However, this can be overruled by defining `sequential=False` in the declaration.. i.e.
```
    sequential xtriggers default = True
    [[xtriggers]]
        clock_1 = wall_clock(sequential=False)
```
or to overwrite the function default
```
    [[xtriggers]]
        clock_1 = wall_clock(sequential=False)
```
(so declaration trumps all)
This respects local trumps global..

For wall_clock, there is no downside of having `sequential=True` the default (with time itself being sequential, and spawning will be nigh instant for historical runs (out to RH))..

Reply:
Hmm.. On second thought, it might be clearer if workflow or declaration overwrites the function default   (but not the undeclared  `sequential xtriggers default` if I can distinguish them).. Will have a look

Reply:
No, in two minds .. I think if it's defined in the xtrigger function, then it's intentional no matter the workflow-wide default, and should only be explicitly overwritten in the xtrigger declaration